pom.xml

```diff
@@ -282,14 +282,13 @@
         <gravitee-policy-graphql-rate-limit.version>1.0.1</gravitee-policy-graphql-rate-limit.version>
         <gravitee-resource-schema-registry-confluent.version>3.1.0</gravitee-resource-schema-registry-confluent.version>
         <gravitee-reactor-message.version>5.0.0-alpha.11</gravitee-reactor-message.version>
-        <gravitee-reactor-native-kafka.version>1.0.0-alpha.37</gravitee-reactor-native-kafka.version>
+        <gravitee-reactor-native-kafka.version>1.0.0-alpha.38</gravitee-reactor-native-kafka.version>
         <gravitee-apim-repository-bridge.version>5.1.2</gravitee-apim-repository-bridge.version>
-        <gravitee-secretprovider-hc-vault.version>2.0.0-alpha.4</gravitee-secretprovider-hc-vault.version>
-        <gravitee-secretprovider-aws.version>2.0.0-alpha.2</gravitee-secretprovider-aws.version>
-        <gravitee-service-secrets.version>1.0.0-alpha.4</gravitee-service-secrets.version>
+        <gravitee-secretprovider-hc-vault.version>2.0.0</gravitee-secretprovider-hc-vault.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**         <gravitee-secretprovider-hc-vault.version>*****</gravitee-secretprovider-hc-vault.version>  
[This comment was created by Aqua Pipeline]